src/explore-education-statistics-frontend/src/modules/download/DownloadIndexPage.tsx

```diff
@@ -57,7 +57,10 @@ const DownloadIndexPage: NextPage<Props> = ({ themes = [] }) => {
       </div>
 
       {themes.length > 0 ? (
-        <Accordion id="downloads">
+        <AccordionWithAnalytics
```

Comment:
Let's get rid of `AccordionWithAnalytics`. There's actually no need for it as it doesn't do anything that can't be achieved just by using the Accordion's `onToggle` prop.